src/models/item.py

```diff
@@ -55,6 +59,9 @@ def __init__(self, data: dict):
 
         self.scanned_on = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
 
+        if (config.locale and not config.locale.startswith('en')):
+            humanize.i18n.activate(config.locale)
```

Comment:
Could we move this part (without the if condition) to the end of the `Config.__init__()`? I think this only needs to be done once and looks quite expensive ...
This way it would also not be necessary to hand the config to the item object.

Alternatively only pass the locale string (`en_US`) to the item?

Reply:
Thanks for your feedback. I've move this part into `Config.__init__()`, it makes more sense.

However, it is necessary to keep the condition because the humanize library does not have the default English language and also produces an error if nothing is passed to it:

```
Traceback (most recent call last):
  File "/home/app/src/main.py", line 279, in <module>
    main()
  File "/home/app/src/main.py", line 124, in main
    config = Config(config_file) if path.isfile(config_file) else Config()
  File "/home/app/src/models/config.py", line 143, in __init__
    humanize.i18n.activate(self.locale)
  File "/usr/local/lib/python3.10/site-packages/humanize/i18n.py", line 73, in activate
    translation = gettext_module.translation("humanize", path, [locale])
  File "/usr/local/lib/python3.10/gettext.py", line 592, in translation
    raise FileNotFoundError(ENOENT,
FileNotFoundError: [Errno 2] No translation file found for domain: 'humanize'
```

Tell me if you have a better idea

Reply:
OK, I see. That's somehow silly. Why did they not explicitly support the english language or set it as default if `None` is provided?

Reply:
Good question. I will be looking to open an issue or PR on their project. I didn't check if another library did the same thing...